src/personnummer.js

```diff
@@ -84,6 +76,23 @@ const getParts = (ssn) => {
     century = ('' + (baseYear - ((baseYear - year) % 100))).substr(0, 2);
   }
 
+  if (sep !== '-' && sep !== '+') {
+    if ((new Date).getFullYear() - parseInt(century + year, 10) < 100) {
+      sep = '-';
+    } else {
+      sep = '+';
+    }
+  } else {
+    // Set the right separator to match the full year.
+    // >= 100 should use + and < 100 should use -
+    const yearDiff = (new Date().getFullYear()) - parseInt(century + year, 10);
```

Comment:
I think we could ignore the if here really. Just assign sep without checking what it was before, as it doesn't really matter :)